xbmc/video/VideoDatabase.cpp

```diff
@@ -4602,11 +4617,43 @@ void CVideoDatabase::UpdateTables(int iVersion)
     m_pDS->exec("ALTER TABLE tvshow ADD userrating integer");
     m_pDS->exec("ALTER TABLE musicvideo ADD userrating integer");
   }
+
+  if (iVersion < 97)
+  {
+    m_pDS->exec("ALTER TABLE movie ADD votes integer");
+    m_pDS->exec("ALTER TABLE episode ADD votes integer");
+    m_pDS->exec("ALTER TABLE tvshow ADD votes integer");
+    m_pDS->exec("ALTER TABLE musicvideo ADD votes integer");
+
+    std::string sql = PrepareSQL("select idMovie,cc%02d from movie", VIDEODB_ID_VOTES);
+    m_pDS->query(sql);
+    while (!m_pDS->eof())
+    {
+      m_pDS->exec(PrepareSQL("UPDATE movie SET votes=%i WHERE idMovie=%i", StringUtils::ReturnDigits(m_pDS->fv(1).get_asString()), m_pDS->fv(0).get_asInt()));
```

Comment:
You can't use `m_pDS` to do something while iterating over a query result of `m_pDS`. Use `m_pDS2`.